README.md

```diff
@@ -6,9 +6,27 @@ We want to raise awareness of environmental sustainability as a key element of o
 
 The [Environmental Sustainability](https://github.com/cncf/env-sustainability-wg/charter.md) WG Charter further outlines the scope of our group activities as well as the goals and intended deliverables.
 
+## Governance
+
+The Environmental Sustainability [charter](charter.md) outlines the scope of our group activities, as part of our [governance process](/governance/README.md) which details how this group works together.
+
 ## Meetings
 
-* The working group meets every Wednesday at 5pm CEST/GMT+2 on [Zoom](https://zoom.us/my/cncfwgenvsustainability)
+* The working group meets every Wednesday at 5pm CEST/GMT+2 or 8am PST
+  * Meeting Link: https://zoom.us/my/cncfwgenvsustainability
+  * Password: 77777
 * Meeting notes & agenda can be found [here](https://docs.google.com/document/d/1TkmMyXJABC66NfYmivnh7z8Y_vpq9f9foaOuDVQS_Lo/edit#)
 * Public discussion runs on https://lists.cncf.io/g/cncf-wg-env-sustainability/ via mailto:cncf-wg-env-sustainability@lists.cncf.io
 * Reach us at [#wg-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6) slack channel on slack.cncf.io
+
+## Members
+
+> The WG is currently in its initialization phase, members need to be elected, projects needs to be organized and roles assigned
+
+### Environmental Sustainability Chairs
```

Comment:
Can we add an extra chair from a different org so that we minimize any potential conflicts of interests?

Reply:
+1

Reply:
echo that and want to put forward a team selection process so that we don't hit the scaling limit. I'd like this process based on
- Availability. The selected members should have cycles for the designated community services 
- Capability. They should be capable of doing what the community expects
- Diversity. They represent diverse backgrounds, geolocations, and affiliations.

Reply:
Echoing the concerns on diversity both of companies and of people.

Reply:
a quorum is intended - so ideally we will have like 3 Co-Chairs and min. 2 TLs

also would like to highlight that the TL is considered part of the leadership team

Reply:
TL as in Team Lead?

Reply:
> TL as in Team Lead?

yes

Reply:
I would propose 3 TLs. The TL team should sufficiently staffed to cover an entire cycle of sustainability research, practice, and end user adoption. 

- The sustainability researcher's role as a TL is to identify scientific research, methodology, and future directions. I would nominate @wangchen615 for this role.
- The practitioner's role as a TL is to use these research and methodology to integrate with CNCF ecosystem and projects. The practitioner must have track record of CNCF contribution and engagement. I would recommend @williamcaban
- The end user's role as a TL is to adopt the projects and best practices, provide feedback for enhancement and directions.

Reply:
> > TL as in Team Lead?
> 
> yes

Actually Tech Lead